### DISCUSSION TITLE
Need some urgent help regarding domains nameserver

### DISCUSSION THREAD

**Discussion by ARITRA69** [vercel/community#3884] [Help]
### Vercel is not retrieving the nameservers of my domain. 
I have set my nameervers as 
```ns1.vercel-dns.com```
```ns2.vercel-dns.com```
in my domain.
But vercel is showing an error in current nameervers section ```No Nameservers Found```

### Steps to Reproduce

.

**Comment by amyegan** [comment]
Hey @ARITRA69. We have a doc with some troubleshooting steps to help with this: https://vercel.com/docs/projects/domains/troubleshooting#

If you share the domain, I can help you look into what is going wrong.

**Reply by ARITRA69** [reply]
aritra.design 
I brought it from webspacekit. I have also set all cname, anames and nameservers of vercel in that domain. please help me :")

**Reply by amyegan** [reply]
I don't see any A record or nameservers coming through for this domain. 
- A record: https://www.whatsmydns.net/#A/aritra.design
- Nameservers: https://www.whatsmydns.net/#NS/aritra.design
 
Please reach out to [WebSpacekit support](https://webspacekit.com/client/submitticket.php?step=2&deptid=5) for help. They should be able to help you figure out why the records are not propagating from their servers.

**Reply by ARITRA69** [reply]
thank you. I'll contact them asap
